src/main/views/idam-signin.njk

```diff
@@ -0,0 +1,54 @@
+{% from "./macros/common-components.njk" import goBack %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "IDAM SignIn" | safe }}
+{% endblock %}
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();event.preventDefault();"
+    }
+  }) }}
+{% endblock %}
+{% block content %}
+  <form method="post" novalidate>
+    <h1 class="govuk-heading-l govuk-!-margin-top-5">Sign in to your account</h1>
+    <div class="govuk-body">You need to <a class="govuk-link" href="#">create an account</a> to use this service.</div>
+    <div class="govuk-body">Already have an account? Please tell us which jurisdiction you are signed up with and we will then ask you to sign in.</div>
+    <div class="govuk-grid-row">
+    {{ govukSelect({
+      id: "idam-select",
+      name: "idam-select",
+      classes: "govuk-grid-column-full",
+      label: {
+        text: "What service jurisdiction is your organisation set up as?"
+      },
+      items: [
+        {
+          value: "",
+          text: ""
+        },
+        {
+          value: "crime",
+          text: "Crime"
+        },
+        {
+          value: "cft",
+          text: "CFT"
+        }
+      ]
+    }) }}
+    </div>
+    {{ govukButton({
+      text: "Continue",
+      classes: "govuk-!-margin-top-5 govuk-!-margin-bottom-8"
+    }) }}
+  </form>
+
+{% endblock %}
```

Comment:
I would add javascript at the bottom of the page which will preventDefault if nothing is selected. With this you avoid redirects for each continue click and it doesn't add record to the browsers navigation history (back won't take you to the same page again for each click)

Reply:
I've done something similar in the search-option.njk file

Reply:
javascript added

Reply:
can you make it === please